streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractDualSchemaRocksDBSegmentedBytesStoreTest.java

```diff
@@ -157,43 +157,34 @@ public void shouldPutAndFetch() {
         try (final KeyValueIterator<Bytes, byte[]> values = bytesStore.fetch(
             Bytes.wrap(keyA.getBytes()), 0, windows[2].start())) {
 
-            final List<KeyValue<Windowed<String>, Long>> expected = asList(
-                KeyValue.pair(new Windowed<>(keyA, windows[0]), 10L),
-                KeyValue.pair(new Windowed<>(keyA, windows[1]), 50L)
-            );
+            // all records expired as actual from is 59001 and to is 1000
```

Comment:
Could you make it clear that where the 59001 come from? (I think it's observed time 60000 - retention 1000 + 1, but please make it clear in comment). Although we can understand the emptyIterator is because records are expired, we are unclear where the 59001 come from. Thanks.

Reply:
Any thoughts for the comment?

Reply:
Oh yeah, these I have addressed. I didn't push the changes yet as I thought there might be some more changes needed after the other comment about test case failure. i will push them soon

Reply:
pushed another commit with the requisite comments.